projects/kit/src/lib/masks/number/utils/tests/stringify-number.spec.ts

```diff
@@ -0,0 +1,385 @@
+import {describe, expect, it} from '@jest/globals';
+
+import {
+    CHAR_EM_DASH,
+    CHAR_EN_DASH,
+    CHAR_HYPHEN,
+    CHAR_JP_HYPHEN,
+    CHAR_MINUS,
+} from '../../../../constants';
+import {maskitoStringifyNumber} from '../stringify-number';
+
+describe('maskitoStringifyNumber', () => {
+    describe('decimal separator is dot (default one)', () => {
+        it('thousand separator is space', () => {
+            expect(
+                maskitoStringifyNumber(1000000.42, {
+                    precision: 2,
+                    decimalSeparator: '.',
+                    thousandSeparator: ' ',
+                }),
+            ).toBe('1 000 000.42');
+        });
+
+        it('thousand separator is hyphen', () => {
+            expect(
+                maskitoStringifyNumber(1000000.42, {
+                    precision: 2,
+                    decimalSeparator: '.',
+                    thousandSeparator: '-',
+                }),
+            ).toBe('1-000-000.42');
+        });
+
+        it('thousand separator is empty string', () => {
+            expect(
+                maskitoStringifyNumber(1000000.42, {
+                    precision: 2,
+                    decimalSeparator: '.',
+                }),
```

Comment:
Empty string is not default value for `thousandSeparator`

https://github.com/taiga-family/maskito/blob/965d7358ad39888d3844c121dd6934ee66cdc541/projects/kit/src/lib/masks/number/number-mask.ts#L38-L42

Reply:
solved